src/main/java/se/sundsvall/myrepresentative/api/model/MandateTemplate.java

```diff
@@ -0,0 +1,25 @@
+package se.sundsvall.myrepresentative.api.model;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder(setterPrefix = "with")
+@Schema(description = "Base information for the issuer of mandates / authorities if provided, 'partyId' and 'type' must be set.")
```

Comment:
Wrong comment? no partyId or type in this class, or am I missing something? :)

Reply:
You caught me copy-pasting perhaps.